linera-service/src/linera.rs

```diff
@@ -1026,6 +1026,10 @@ enum NetCommand {
         /// TESTING ONLY.
         #[structopt(long)]
         testing_prng_seed: Option<u64>,
+
+        /// The name for the database table to store the chain data in.
+        #[structopt(long)]
```

Comment:
Please add a default value.

Reply:
Or perhaps better: make it optional? (because it's ignored for rocksdb isn't ?)

Reply:
Ok, done.